### ISSUE TITLE
Python.Deployment Deploy other versions of Python Not in effect？

### ISSUE THREAD

**Issue by pawn86** [henon/Python.Included#58]
Thank you very much for your clarification.
Install python 11.3 using Python.Deployment (I deployed 11.
2 versions), but ended up sys.version=11.2 instead of the expected 11.3.
![issur](https://github.com/henon/Python.Included/assets/69451647/c5c06309-7516-4f12-8d14-4df2a5709eee)

**Comment by henon** [comment]
And did you embed 
![image](https://github.com/henon/Python.Included/assets/44090/80977196-73da-4f0d-843f-ef2625c78f9d)
in your assembly?

What is the other version it loads, did you install it manually? 

- Make sure the install location is empty. 
- Make sure it installs the version you want by checking the files in the install location and seeing if it is really the right version.
- Make sure it doesn't load another python version from a different location instead. If so, you probably need to set the pythonnet PythonPath or something like that.

**Comment by pawn86** [comment]
Good idea, it's been worked out. The python I manually installed before was 3.11.2. I used the pythonnet library directly and set the environment variable PYTHON_PYDLL. This is the problem caused by this environment variable. Solved my problem perfectly after deletion. Thank you so much.
